### DISCUSSION TITLE
Error 403 with nginx root /usr/local/searx/searx-src/searx;

### DISCUSSION THREAD

**Discussion by jjazar** [searx/searx#2333] [Installation issue: Official wiki]
(/etc/nginx/sites-available/searx)

I'm on debian 10 and installed via scripts (after an unsuccessful manual install) and I believe everything is working fine, besides the fact that the site won't show. 

I can use /var/www/index.html or whatever if I make one and it will work, but not if I set the root as searx. The old guide had the root be set to /usr/local/searx/searx, but that directory seems to be equivalent to what is now /usr/local/searx/searx-src/searx - however, this just returns a 403 error. I don't see any documentation on this - any tips?

**Comment by unixfox** [comment]
Did you install using the official installation scripts?

**Comment by jjazar** [comment]
The ones on the wiki, yes. It was left as the default, /var/ww/htm(l?). I'll retry later with a fresh install and see if it happens again.

**Comment by unixfox** [comment]
If you are having some difficulties for deploying Searx you should try searx-docker: https://github.com/searx/searx-docker. It makes the job easier of deploying Searx on a server.

**Comment by return42** [comment]
> The ones on the wiki, 

Which wiki? Read [installation instructions](https://searx.github.io/searx/admin/installation.html).

**Comment by chrisJuresh** [comment]
Im having the same issue
